packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -33,7 +33,7 @@ export interface AuthFlow {
 
   /**
    * Enable authflow to refresh tokens
-   * @default false
+   * @default true
```

Comment:
is this a breaking change? - flipping the behaviour from false to true?

potential follow up question: if it must always be enabled, is there any value in accepting it as a boolean

Reply:
This should not be a breaking change since, as pointed out in the issue, this flag is anyway required if we are providing authFlows.

There is no specific point of accepting it as boolean. But removing it would then be a breaking change. What do you think?

Reply:
Removed the prop and marking this change as breaking